From nobody@FreeBSD.ORG Mon Sep 20 03:45:51 1999
Return-Path: <nobody@FreeBSD.ORG>
Received: by hub.freebsd.org (Postfix, from userid 32767)
	id 7CABF15107; Mon, 20 Sep 1999 03:45:51 -0700 (PDT)
Message-Id: <19990920104551.7CABF15107@hub.freebsd.org>
Date: Mon, 20 Sep 1999 03:45:51 -0700 (PDT)
From: sobomax@altavista.net
Sender: nobody@FreeBSD.ORG
To: freebsd-gnats-submit@freebsd.org
Subject: [PATCH] Empty "Themes" menu in wmakerconf.
X-Send-Pr-Version: www-1.0

>Number:         13854
>Category:       ports
>Synopsis:       [PATCH] Empty "Themes" menu in wmakerconf.
>Confidential:   no
>Severity:       non-critical
>Priority:       low
>Responsible:    cpiazza
>State:          open
>Quarter:        
>Keywords:       
>Date-Required:  
>Class:          change-request
>Submitter-Id:   current-users
>Arrival-Date:   Mon Sep 20 03:50:01 PDT 1999
>Closed-Date:    
>Last-Modified:  Tue Sep 21 09:35:13 PDT 1999
>Originator:     Maxim Sobolev
>Release:        4.0-CURRENT
>Organization:
Vega International Capital
>Environment:
FreeBSD notebook 4.0-CURRENT FreeBSD 4.0-CURRENT #3: Thu Sep  2 18:13:40 EEST 1999     root@notebook:/usr/src/sys/compile/NOTEBOOK  i386

>Description:
Wmakerconf has empty "Themes" menu probably due "work in progress". Following is the patch (patch-aa) to fix this misbehaviour.
>How-To-Repeat:
$ wmakerconf
>Fix:
--- src/menu.c.orig	Mon Sep 20 13:40:07 1999
+++ src/menu.c	Mon Sep 20 13:40:47 1999
@@ -202,9 +202,11 @@
    gtk_menu_item_set_submenu (GTK_MENU_ITEM (item), file_menu);
    gtk_menu_bar_append (GTK_MENU_BAR (menu_bar), item);
 
+#if defined(NETSCAPE)
    item = gtk_menu_item_new_with_label (_("Themes"));
    gtk_menu_item_set_submenu (GTK_MENU_ITEM (item), themes_menu);
    gtk_menu_bar_append (GTK_MENU_BAR (menu_bar), item);
+#endif /* defined(NETSCAPE) */
 
    item = gtk_menu_item_new_with_label (_("Help"));
    gtk_menu_item_set_submenu (GTK_MENU_ITEM (item), help_menu);


>Release-Note:
>Audit-Trail:
State-Changed-From-To: open->closed  
State-Changed-By: cpiazza 
State-Changed-When: Mon Sep 20 17:51:49 PDT 1999 
State-Changed-Why:  
This was removed in 0.61.0 and things like this are better submitted 
to the program authors for actively developed programs anyway 
State-Changed-From-To: closed->open  
State-Changed-By: cpiazza 
State-Changed-When: Tue Sep 21 09:34:38 PDT 1999 
State-Changed-Why:  
Woops, This shouldn't have been closed 


Responsible-Changed-From-To: freebsd-ports->cpiazza  
Responsible-Changed-By: cpiazza 
Responsible-Changed-When: Tue Sep 21 09:34:38 PDT 1999 
Responsible-Changed-Why:  
My mistake, I'll handle this 
>Unformatted:

